backend/src/sfu.rs

```diff
@@ -604,11 +604,16 @@ impl Sfu {
 
                     time_scope_us!("calling.sfu.handle_packet.rtcp.in_outgoing_connection_lock");
 
-                    if let Some(key_frame_request) =
-                        outgoing_connection.send_key_frame_request(key_frame_request)
-                    {
+                    let key_frame_res =
+                        outgoing_connection.send_key_frame_request(key_frame_request);
+
+                    if let Ok(key_frame_request) = key_frame_res {
```

Comment:
If you keep the `key_frame_res` variable, it should be named `key_frame_result`, we tend to spell things out (usually), but when it is next to `key_frame_request` it just looks wrong.

However, I don't see why we need the variable, can we keep it like before? Or better yet, wouldn't this block be better now as a `match` statement?

Reply:
yeah I agree with you sir, match would be syntactically appropiate to place here, Thanks I will refactor accordingly.